### PR TITLE
/docs/integrations headers

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request applies consistency and standardization to `/docs/integrations` headers so that they're always plural.

## Select the type of change

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/sumo-apps/enterprise-audit.md [hunk 1]
```diff
@@ -62,14 +62,14 @@ import ViewDashboards from '../../reuse/apps/view-dashboards.md';
 
 <img src={useBaseUrl('img/integrations/sumo-apps/EA_Collector_Data_Forwarding_icon.png')} alt="test" />
 
-Enterprise Audit - Collector and Data Forwarding Management App dashboards present information on Collector, Sources activities, and data forwarding trends by destination types. This App has the following two dashboard categories:
+The **Enterprise Audit - Collector and Data Forwarding Management App** dashboards present information on Collector, Sources activities, and data forwarding trends by destination types. This App has the following two dashboard categories:
```
Comment (jpipkin1) [inline docs/integrations/sumo-apps/enterprise-audit.md:65]:
```suggestion
The **Enterprise Audit - Collector and Data Forwarding Management App** dashboards present information on Collector, Sources activities, and data forwarding trends by destination types. This app has the following two dashboard categories:
```

File: docs/send-data/installed-collectors/configuration.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ As the name implies, an Installed Collector is installed in your environment, as
 
 A Sumo Source is an object configured for a specific Collector that sends data to Sumo Logic. There are a number of Source types that work with Installed Collectors. For a list of all Sources supported by Installed Collectors, see [Sources for Installed Collectors](/docs/send-data/installed-collectors/sources).
 
-See [How to choose a collector](/docs/send-data/choose-collector-source) for guidance on when to use a single or multiple Installed Collectors.
+See [How to Choose a Collector](/docs/send-data/choose-collector-source) for guidance on when to use a single or multiple Installed Collectors.
```
Comment (jpipkin1) [inline docs/send-data/installed-collectors/configuration.md:13]:
```suggestion
See [Choosing a Sumo Logic Collector and Source](/docs/send-data/choose-collector-source) for guidance on when to use a single or multiple Installed Collectors.
```

**Review by jpipkin1 — APPROVED** [review]
Looks good, @kimsauce! I have a couple of suggestions.

**Comment by kimsauce** [comment]
> Looks good, @kimsauce! I have a couple of suggestions.

Thanks for the quick review!

**Pull request merged**